test/spell-checker.js

```diff
@@ -78,6 +78,13 @@ eslintTester.addRuleTest('rules/spell-checker', {
             errors: [
                 { message: 'You have a misspelled word: tsih on String'},
                 { message: 'You have a misspelled word: srting on String'}]
+        },
+        {
+            code: 'var a = 1 // color is a comment normalize dict',
+            args:[2, {lang: 'en_GB', skipWords: ['dict']}],
+            errors: [
+                { message: 'You have a misspelled word: color on Comment'},
+                { message: 'You have a misspelled word: normalize on Comment'}]
```

Comment:
Sure will do - this is clearly why I need a spell checker :) 

Also is there a way for me to run the tests locally?